### PR TITLE
Added Pic2Word notebook to Model Garden github

### PR DESCRIPTION
**Pull request by jismailyan-google**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
--- YOUR PR SUMMARY GOES HERE ---
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb [hunk 1]
```diff
@@ -0,0 +1,467 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: <p>workbench -&gt; Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_ID = "automl-migration-test"
</pre><p>PROJECT_ID = ["your-project-id"]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: <p>Vertex Model Registry -&gt; Vertex AI Model Registry</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.        project="automl-migration-test",
</pre><p>projectt="..." -&gt; project=PROJECT_ID</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.        location="us-central1",
</pre><p>location=REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: <p>Add cleanup section to end of notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (jismailyan-google) [reply notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: Done

Comment (jismailyan-google) [reply notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: Done

Comment (jismailyan-google) [reply notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: Done

Comment (jismailyan-google) [reply notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: Done

Comment (jismailyan-google) [reply notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb:1]: Done

**Pull request merged**